### DISCUSSION TITLE
Chainlink DECO

### DISCUSSION THREAD

**Discussion by irux** [tlsnotary/tlsn#140] [General]
Hello!

I find this super interesting. Do you know about DECO from chainlink? What would be the difference from it?

PS: do you have something (papers, tutorials, background info) I can read to understand it more? I find the concept amazing!

**Comment by themighty1** [comment]
Hi, feel free to join our Discord (the "Chat with us" link at the bottom of tlsnotary.org)
Someone asked there about DECO in this thread https://discord.com/channels/974410303255232542/974410303255232545/1046021991028768828
